src/components/SectionTable/CourseInfoBar.js

```diff
@@ -138,8 +138,8 @@ class CourseInfoBar extends PureComponent {
                     }}
                     style={{ marginRight: '4px' }}
                 >
+                    <InfoOutlinedIcon fontSize="small" marginRight={5} />
```

Comment:
![image](https://user-images.githubusercontent.com/29494270/159186006-7711d145-f32b-4864-b702-bcfe1af1c60a.png)

Doesn't look like the `marginRight` is working for me. Let's do an inline style instead.
```suggestion
                    <InfoOutlinedIcon fontSize="small" style={{ marginRight: '5px' }} />
```